core/src/main/scala/kafka/log/LogValidator.scala

```diff
@@ -175,12 +175,6 @@ private[kafka] object LogValidator extends Logging {
                                                  messageFormatVersion: Byte = RecordBatch.CURRENT_MAGIC_VALUE,
                                                  messageTimestampType: TimestampType,
                                                  messageTimestampDiffMaxMs: Long): ValidationAndOffsetAssignResult = {
-      // Deal with compressed messages
```

Comment:
Some of the comments below still reference these steps. Should we remove them also?